frontend/src/__mocks__/mockPVCK8sResource.ts

```diff
@@ -0,0 +1,138 @@
+import { PersistentVolumeClaimKind } from '~/k8sTypes';
+
+type MockResourceConfigType = {
+  name?: string;
+  namespace?: string;
+  storage?: string;
+};
+
+export const mockPVCK8sResource = ({
+  name = 'test-storage',
+  namespace = 'test-project',
+  storage = '5Gi',
+}: MockResourceConfigType): PersistentVolumeClaimKind => ({
+  kind: 'PersistentVolumeClaim',
+  apiVersion: 'v1',
+  metadata: {
+    annotations: {
+      'openshift.io/description': '',
+      'openshift.io/display-name': 'Test Storage',
+      'pv.kubernetes.io/bind-completed': 'yes',
+      'pv.kubernetes.io/bound-by-controller': 'yes',
+      'volume.beta.kubernetes.io/storage-provisioner': 'ebs.csi.aws.com',
+      'volume.kubernetes.io/selected-node': 'ip-10-0-133-119.us-west-2.compute.internal',
+      'volume.kubernetes.io/storage-provisioner': 'ebs.csi.aws.com',
+    },
+    resourceVersion: '864084751',
+    name,
+    uid: '8644e33b-a710-45a3-9d54-7f987494643a',
+    creationTimestamp: '2023-03-03T03:17:32Z',
+    managedFields: [
```

Comment:
Just a nit pick, do we need managedFields for mocks? It's not bad, but it's an internal resource that I think it has no value for us, just curious what the rest think about it @andrewballantyne @alexcreasy @Gkrumbach07